### ISSUE TITLE
BUG: Enemies at Cavalry Hill take 1 damage from too far away on spawn

### ISSUE THREAD

**Issue by EmosewaMC** [DarkflameUniverse/DarkflameServer#1127]
### Make sure you've done the following:

- [X] I have checked that this issue has not already been reported.

- [X] I have validated that this issue is not a syntax error of either MySQL or SQLite.

- [X] I have pulled the latest version of the main branch of DarkflameServer and have confirmed that the issue exists there.

### DarkflameServer Version

41898be

### Issue Description

Title
![image](https://github.com/DarkflameUniverse/DarkflameServer/assets/39972741/60abb727-e8c8-4cb7-ae7e-3acaa1aefd11)

### Reproduction Steps

1) spawn an enemy at cavalry hill
2) observe that they take 1 damage.

### Expected Behavior

They should not take 1 damage on spawn

### Environment

WSL

**Comment by Terrev** [comment]
just noting that this happens at the fallen gate as well (embedded screenshot above is of that location). screenshot is of basic ag stromling (1hp) dying upon getting close to crypt

**Comment by Terrev** [comment]
quick note - while spawning/testing some enemies in lego club i noticed one of the rings closest to the ground on the right side (assuming youre coming in through the front door and facing that way) was getting smashed, and dropping no loot. it seemed like the enemy may have been tripping the collision with it even though they were very far apart, vertically.

looking at the stromling getting smashed in the above screenshot, it looks like something similar may be going on; theres no way the actual collision box thats doing the damage should be reaching remotely that far, even with current AABB behavior

tl;dr issue may be twofold:
- enemies tripping collision serverside from way, way, way too far away in general
- enemies PROBABLY shouldnt be taking damage from those specific trigger boxes in fv (though if the collision were happening properly, this bug actually wouldnt be observable at all unless you really tried to make it happen - for all we know, them taking damage from these IS how it was in live and we just never knew because enemies would never usually enter the collision volume there)

**Comment by aronwk-aaron** [comment]
This is an issue with the aggro physics radius colliding and causing the enemy to take damage
